src/api/deploy/helpers/schema/deploy-service-validation.js

```diff
@@ -8,23 +8,63 @@ const validCpuValues = Object.keys(ecsCpuToMemoryOptionsMap).map((cpu) =>
   Number.parseInt(cpu)
 )
 
-const memoryValidation = buildMemoryValidation()
+const memoryValidation = buildMemoryValidation().required()
```

Comment:
nit: is there value for this abstraction? For me this makes this harder to read, considering in the object you have the key which names the validation